app/views/admin/users/edit.html.erb

```diff
@@ -21,7 +21,7 @@
       <h2>
         User Status
         <% if @user.banned %>
-          <span class="badge badge-danger">🚨 Member is Suspended 🚨</span>
+          <span class="badge badge-danger">🚨 Member is Banned 🚨</span>
```

Comment:
We display "suspended" as "banned" in a user's role history, so it made sense to switch this to also display as "banned" rather than "suspended" for consistency purposes!